pyproject.toml

```diff
@@ -8,12 +8,24 @@ version = "5.0.0"
 authors = [{name = "Sam Schott", email = "sam.schott@outlook.com"}]
 license = {text = "MIT"}
 description = "Python library for cross-platform desktop notifications"
-keywords = ["desktop-notifier"]
+keywords = [
+    "D-Bus",
+    "Desktop Bus",
```

Comment:
IMO, D-Bus and Desktop Bus are slightly misleading keywords which may suggest that this is a some kind of dbus library, whereas desktop notifier just uses it as a communication channel.

Reply:
happy to delete it; whatever you think. my thinking was just that someone who is familiar with linux enough to be using a notifications app would theoretically know that such a library would have to use D-Bus? but tbh i don't know anything about D-Bus (literally never heard of it) or really any kind of linux desktop environment.

Reply:
Yeah, I think it's better to remove those two.